app/src/main/java/org/mozilla/fenix/compose/Favicon.kt

```diff
@@ -23,42 +22,49 @@ import mozilla.components.browser.icons.IconRequest
 import mozilla.components.browser.icons.compose.Loader
 import mozilla.components.browser.icons.compose.Placeholder
 import mozilla.components.browser.icons.compose.WithIcon
-import mozilla.components.ui.colors.PhotonColors
 import org.mozilla.fenix.components.components
 import org.mozilla.fenix.theme.FirefoxTheme
 import org.mozilla.fenix.theme.Theme
 
 /**
  * Load and display the favicon of a particular website.
  *
- * @param url Website [URL] for which the favicon will be shown.
+ * @param url Website [url] for which the favicon will be shown.
  * @param size [Dp] height and width of the image to be loaded.
+ * @param modifier [Modifier] allowing to control among others the dimensions and shape of the image.
  * @param isPrivate Whether or not a private request (like in private browsing) should be used to
  * download the icon (if needed).
  */
 @Composable
 fun Favicon(
     url: String,
     size: Dp,
-    isPrivate: Boolean = false
+    modifier: Modifier = Modifier,
```

Comment:
Let's make `modifier` the last parameter. I am partly influenced by what I saw in [the blog](https://chris.banes.dev/always-provide-a-modifier/).

Reply:
I actually got an Android Studio warning when putting it not as the first optional parameter, but I'm happy to make it the last still.

Reply:
I see the same error. Doesn't seem like it's from detekt so no need to change it then.

Reply:
@MozillaNoah That is because you provided a default value for it, so it has to be the first optional param. There are some different use cases regarding chaining that need this. But one of the other reasons for this convention:  as the modifier is the first optional param you can always provide a value for it without using named parameters.